### PR TITLE
Bug: Fix compatibility running ecFSEOF with GECKO3 models

### PR DESCRIPTION
**Pull request by ae-tafur**
### Main improvements in this PR:

- Fix compatibility in ecFSEOF to GECKO3 models
- Change name function `simulateGrowth` to follow #229 
- Model adapter class was added to use `userData/model/output` as default folder to store results

**I hereby confirm that I have:**

- [ ] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m [hunk 1]
```diff
@@ -1,53 +1,72 @@
...
+% for a specified production target.
 %
-% Usage: run_ecFSEOF(ecModel,rxnTarget,cSource,alphaLims,Nsteps)
+% Input:
+%   ecModel         an ecModel in GECKO 3 format (with ecModel.ec structure).
+%   rxnTarget       rxn ID for the production target reaction, a exchange
+%                   reaction is recommended.
+%   cSource         rxn ID for the main carbon source uptake reaction.
+%   alphaLims       vector of Minimum and maximum biomass yield [gDW/mmol Csource]
+%                   for enforced objective limits (e.g. [0.5 1]). Max value: 1.
```
Comment (edkerk) [inline src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m:13]: I don't understand why the maximum value should be 1?

Comment (ae-tafur) [reply src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m:13]: Alpha is scaling factor for desired suboptimal growth (which fix the LB of biomass reaction), then a value higher than 1 will result in non feasible solutions. e.g if the maximum growth is 0.41 an scaling factor of 1.1 mean fix the LB to 0.451, a value that cannot be reached.

Comment (edkerk) [reply src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m:13]: Ok, but then the description is somewhat confusing. I thought that these values are gDW/mmol Csource, but they are actually scaling factors of the experimental growth rate.

Comment (ae-tafur) [reply src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m:13]: Yeap, you are right I will change it

File: src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m [hunk 2]
```diff
@@ -1,53 +1,72 @@
...
 %
-% Usage: run_ecFSEOF(ecModel,rxnTarget,cSource,alphaLims,Nsteps)
+% Input:
+%   ecModel         an ecModel in GECKO 3 format (with ecModel.ec structure).
+%   rxnTarget       rxn ID for the production target reaction, a exchange
+%                   reaction is recommended.
+%   cSource         rxn ID for the main carbon source uptake reaction.
+% 	alphaLims       vector of Minimum and maximum biomass scalling factors for
+%			        enforced objective limits (e.g. [0.5 1]). Max value: 1.
+%   Nsteps          number of steps for suboptimal objective in FSEOF.
```
Comment (edkerk) [inline src/geckomat/utilities/ecFSEOF/run_ecFSEOF.m:14]:
```suggestion
%   cSource         rxn ID for the main carbon source uptake reaction.
%   alphaLims       vector of Minimum and maximum biomass scalling factors for
%                   enforced objective limits (e.g. [0.5 1]). Max value: 1.
%   Nsteps          number of steps for suboptimal objective in FSEOF.
```

**Review by edkerk — APPROVED** [review]

**Pull request merged**